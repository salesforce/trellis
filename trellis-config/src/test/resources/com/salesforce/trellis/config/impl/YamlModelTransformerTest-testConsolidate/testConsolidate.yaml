groups:
- name: FOO_TIER
  includes:
  - foo:foo-*

- name: BAR_TIER
  includes:
  - bar:bar-*

rules:
- action: DENY
  from: [FOO_TIER]
  to: [BAR_TIER]

- action: DENY
  from:
  - foo:baz
  to: [BAR_TIER]

- action: DENY
  from:
  - foo:bing
  to: [BAR_TIER]

- action: WARN
  from:
  - foo:please
  to: [BAR_TIER]
  reason: please dont depend on this
