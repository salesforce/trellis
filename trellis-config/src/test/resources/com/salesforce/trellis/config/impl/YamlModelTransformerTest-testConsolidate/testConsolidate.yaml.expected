groups:

- name: BAR_TIER
  includes:
  - bar:bar-*

- name: FOO_TIER
  includes:
  - foo:foo-*

rules:

- action: DENY
  from:
  - FOO_TIER
  - foo:baz
  - foo:bing
  to:
  - BAR_TIER

- action: WARN
  from:
  - foo:please
  to:
  - BAR_TIER
  reason: please dont depend on this
